content/docs/dvclive/index.md

```diff
@@ -42,6 +42,15 @@ live.log_artifact("model.pt")
 
 See `Live.log_artifact()`.
 
+</tab>
+<tab title="Models">
```

Comment:
Do we need a separate tab? Why not just update the `Artifacts` tab? It's the first one we show, so there will still be an obvious example of logging a model.